src/Core/Resolvers/SqlResponseHelpers.cs

```diff
@@ -0,0 +1,230 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.Text.Json;
+using System.Text.Json.Nodes;
+using Azure.DataApiBuilder.Config.ObjectModel;
+using Azure.DataApiBuilder.Core.Models;
+using Azure.DataApiBuilder.Core.Services;
+using Azure.DataApiBuilder.Service.Exceptions;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Http.Extensions;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Azure.DataApiBuilder.Core.Resolvers
+{
+    public class SqlResponseHelpers
```

Comment:
nit: provide class summary with its purpose. Im assuming this code is just moved from SqlQueryEngine